### ISSUE TITLE
Add proper logging

### ISSUE THREAD

**Issue by om26er** [crossbario/autobahn-java#234]
Right now the only logging that we are doing in the Netty wamp implementation is using System.out.println(); its bad and not really helpful while debugging.

We should replace that with proper Log.* calls.

**Comment by om26er** [comment]
First part of this landed in master, I did not change the demo-gallery bits because that way we lose the fancy printing, we may visit that later.

**Comment by oberstet** [comment]
I think it would be nice to

* expand also the demo gallery for using standard logging
* have the project variables for router URL/realm (as we have in the PoC repo) also in the demo gallery here
* have a new project variable for log level both for demo gallery here, and for the PoC

So essentially, I could then start the demo gallery (and also the PoC) from command line (maybe `-Ploglevel=debug` unless there is a different established convention).

With `loglevel=debug`, eg the library internals would log sent and received WAMP message. With normal loglevel, the library would stay calm, only log on errors or warnings.

To summarize: both using logging inside the library and our examples for actual users of the library should follow Java best practices.

**Comment by om26er** [comment]
The only remaining bit for this issue is to come up with a way to provide the logging level to be used on the command line. I am working on that.

**Comment by oberstet** [comment]
Perfect! I think this really demonstrates that the new ABJ is for serious deployments in backend scenarios. I leave that open, as you mention you want to still work on that last remaining issue.

**Comment by om26er** [comment]
last bits proposed: https://github.com/crossbario/autobahn-java/pull/251

**Comment by om26er** [comment]
final change landed: https://github.com/crossbario/autobahn-java/pull/251
